tests/cppcheck/suppression-list.txt

```diff
@@ -7,6 +7,24 @@ unusedFunction:widgets/glade/*
 // stop whining about G_DEFINE macros
 constStatement:*/widgets/src/*
 constStatement:widgets/src/*
+// The constStatement is replaced with syntaxError in cppcheck 2.16
+// Let's ignore it but be more location specific
```

Comment:
Please add a link to the upstream discussion.

Reply:
Added